From 2e8c58608604b843e896d24f8f69986822a2b4a9 Mon Sep 17 00:00:00 2001
From: danielhk <daniel.p6800@gmail.com>
Date: Sat, 16 Dec 2017 19:42:51 +0800
Subject: [PATCH 4/4] Build flag misplaced

Build flag SUPPRESS_MTK_AUDIO_BLOB_ERR_MSG is for the shared library not
service.
---
 audio/2.0/default/Android.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/audio/2.0/default/Android.mk b/audio/2.0/default/Android.mk
index 84ca6e3..78077ea 100644
--- a/audio/2.0/default/Android.mk
+++ b/audio/2.0/default/Android.mk
@@ -51,6 +51,10 @@ LOCAL_HEADER_LIBRARIES := \
 
 LOCAL_WHOLE_STATIC_LIBRARIES := libmedia_helper
 
+ifeq ($(SUPPRESS_MTK_AUDIO_BLOB_ERR_MSG),true)
+LOCAL_CFLAGS= -DSUPPRESS_MTK_AUDIO_BLOB_ERR_MSG
+endif
+
 include $(BUILD_SHARED_LIBRARY)
 
 #
@@ -88,8 +92,4 @@ ifeq ($(TARGET_USES_BCRADIO_FUTURE_FEATURES),true)
 LOCAL_CFLAGS += -DTARGET_USES_BCRADIO_FUTURE_FEATURES
 endif
 
-ifeq ($(SUPPRESS_MTK_AUDIO_BLOB_ERR_MSG),true)
-LOCAL_CFLAGS += -DSUPPRESS_MTK_AUDIO_BLOB_ERR_MSG
-endif
-
 include $(BUILD_EXECUTABLE)
-- 
2.7.4

